staging/src/k8s.io/apimachinery/pkg/api/meta/restmapper.go

```diff
@@ -62,6 +64,8 @@ type DefaultRESTMapper struct {
 	kindToScope          map[schema.GroupVersionKind]RESTScope
 	singularToPlural     map[schema.GroupVersionResource]schema.GroupVersionResource
 	pluralToSingular     map[schema.GroupVersionResource]schema.GroupVersionResource
+
+	namer namer.Namer
```

Comment:
may give fmt error

Reply:
ACK. What's the fmt tool to check this and how can I trigger it? `gofmt` doesn't report anything